cmd/httpd/handlers/auth.go

```diff
@@ -102,66 +101,46 @@ func (a *Auth) Login(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *Auth) Refresh(w http.ResponseWriter, r *http.Request) {
-	var request validateTokenRequest
-
-	decoder := json.NewDecoder(r.Body)
-	err := decoder.Decode(&request)
-	if err != nil {
-		a.logger.Error().Err(err).Msg("Unable to decode login data")
-
-		w.WriteHeader(http.StatusInternalServerError)
-		return
-	}
+	claims := r.Context().Value("claims").(jwt.MapClaims)
```

Comment:
I'd prefer to store in context `id` and `email` as strings. This will helps us to remove dependency of jwt.MapClaims

Reply:
makes sense

Reply:
fixed